checkout/frames-android
# PIMOB-2053: Integrate SonarQube and add test coverage report task

## Issue

PIMOB-2053

## Proposed changes

* Add `sonar_step` for Bitrise to build the SonarQube report and submit it. 
* Add `jacocoTestReport` task for `:frames` and `:checkout` to build the test coverage report for SonarQube

## Test Steps

1. Push any commit to the repository
2. Verify on Bitrise there's a steps-sonar executed 
3. Verify there's a code analysis and test coverage analysis on SonarQube

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
